main/main.c

```diff
@@ -75,19 +77,18 @@ int ex2_main(int argc, char **argv) {
   InitIO();
 
   /* Initialization routine */
-  //init_filesystem();
+  init_filesystem();
```

Comment:
Would it run fine for people without an SD card? I think it'd be better commented out on the master branch.